grpc/src/test/java/com/linecorp/armeria/client/grpc/GrpcClientBuilderTest.java

```diff
@@ -126,6 +126,16 @@ void enableUnsafeWrapResponseBuffers() {
         assertThat(clientParams.options().get(GrpcClientOptions.UNSAFE_WRAP_RESPONSE_BUFFERS)).isTrue();
     }
 
+    @Test
+    void useMethodMarshaller() {
+        final TestServiceBlockingStub client =
+                GrpcClients.builder("http://foo.com")
+                           .useMethodMarshaller(true)
+                           .build(TestServiceBlockingStub.class);
```

Comment:
Should we add a custom method `Marshaller` to ensure `useMethodMarshaller()` works?
We may override the method marshallers in `ClientInterceptor`.
```java
class MyClientInterceptor implements ClientInterceptor {
  @Override
  public <I, O> ClientCall<I, O> interceptCall(
    MethodDescriptor<I, O> method, CallOptions callOptions, Channel next) {
    
    method.toBuilder().setRequestMarshaller(new Marshaller<I>() {
        @Override
        public InputStream stream(I value) {
            // Add a counter to ensure this method is invoked.
            // Then delegate to method.requestMashaller.stream()
        }

   }
}
```

Reply:
I have checked to ensure `marshaller` in `MethodDescriptor` is working when true for useMethodMarshaller is passed through its constructor like below.

https://github.com/line/armeria/blob/e11b59d3f57d93f60e36aaf21ca456f8ad2a56dd/grpc/src/test/java/com/linecorp/armeria/internal/common/grpc/GrpcMessageMarshallerTest.java#L77-L86

If it's not enough, I'll try to add testcast. 
What do you think?

Reply:
Applied feedback.
Thanks for your recommendation `ClientInterceptor`. It works fine.